bzt/modules/_apiritif/generator.py

```diff
@@ -953,17 +954,26 @@ def _gen_webdriver(self):
         body = [self._get_options(browser)]
 
         if browser == 'firefox':
-            body.extend(self._get_firefox_profile() + [self._get_firefox_webdriver()])
+            if self.selenium_version.startswith("4") and LooseVersion(self.selenium_version) > LooseVersion('4.1.3'):
```

Comment:
why do you need this check 
"self.selenium_version.startswith("4")" ? 
LooseVersion() > 4.1.3 should cover that  I think